### ISSUE TITLE
some inspiration

### ISSUE THREAD

**Issue by balupton** [DamonOehlman/flow-starter#1]
Bevry uses flow quite heavily - our base project https://github.com/bevry/base - example https://github.com/bevry/watchr/blob/fbeeaee072143f08be24087ee7f9483e083dea6f/source/index.js

**Comment by DamonOehlman** [comment]
Awesome - thanks Ben.  I can see you are mostly using the inline comment type notation, which I guess gives you compatibility without transpilation for free.  That's something I've thought about... nice to see a concrete usage example :)

**Comment by balupton** [comment]
Yeah. Until everything uses babel, not doing the comment style breaks too much of one's toolchain - or requires complex intermediary transpilation and configuration steps - which is a pain. Would be great if everything uses babel, even typescript and eslint, but it seems that won't happen.

Using TypeScript has the same issue - beautiful language, but then other parts of one's toolchain get out of sync. Babel is like USB-C - and seems TypeScript and ESLint are Lightning.
